tf2_ros/test/listener_unittest.cpp

```diff
@@ -31,6 +31,7 @@
 #include <gtest/gtest.h>
 #include <tf2_ros/transform_listener.h>
 
+using namespace tf2;
```

Comment:
I'd really prefer not to use `using namespace`; it can confuse the issue on where a symbol is coming from.  I know you didn't add this, but I think it should be easy enough to remove this line and add the `tf2::` as appropriate below.

Reply:
Maybe for the sake of keeping divergence from upstream small keep the `using namespace`?

Reply:
Honestly, though, this branch has diverged so significantly from the melodic-devel branch that I don't think it is worthwhile anymore.  We probably need to maintain them as separate tracks going forward anyway.